stepfunctions/templates/consignment_export_policy.json.tpl

```diff
@@ -0,0 +1,57 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ecs:RunTask"
+      ],
+      "Resource": [
+        "${task_arn}"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ecs:StopTask",
+        "ecs:DescribeTasks"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "events:PutTargets",
+        "events:PutRule",
+        "events:DescribeRule"
+      ],
+      "Resource": [
+        "arn:aws:events:eu-west-2:${account_id}:rule/StepFunctionsGetEventsForECSTaskRule"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iam:PassRole"
+      ],
+      "Resource": [
+        "${task_role}",
+        "${execution_role}"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogDelivery",
```

Comment:
Maybe order alphabetically?

Reply:
I've ordered the three lists in this file.